create_person.rb

```diff
@@ -0,0 +1,29 @@
+class CreatePerson
+  def initialize(people)
+    @people = people
+  end
+
+  def create_person
+    option = get_promt('Do you want to create a new student (1) or a new teacher? (2) [1/2]: ').to_i
+
+    unless [1, 2].include?(option)
+      puts 'Invalid selection'
+      return
+    end
+
+    age = get_promt('Age: ')
+    name = get_promt('Name: ')
+
+    case option
+    when 1
+      permission = get_promt('Has parent permission? [y/n]:') == 'y'
+      student = Student.new(name, 'class', age, parent_permission: permission)
+      @people << student
+    when 2
+      specialization = get_promt('Specialization')
```

Comment:
- [ ] **[OPTIONAL]** I suggest adding `:` when prompting users for `specialization` i.e instead of `specialization = get_promt('Specialization')` let's have `specialization = get_promt('Specialization: ')`. This is because when I was asked for specialization when creating a Teacher, There is no space between my answer and the prompt. See it in the image below:

<img width="146" alt="chrome_u5zyVbyKKn" src="https://user-images.githubusercontent.com/81468675/159687099-46c723d4-7491-4d98-a9c7-64737e082d97.png">